native/c/zowex.cpp

```diff
@@ -858,27 +858,12 @@ int handle_data_set_list(ZCLIResult result)
         cerr << "Warning: no matching results found" << endl;
       }
     }
-    vector<string> fields;
-    for (vector<ZDSEntry>::iterator it = entries.begin(); it != entries.end(); ++it)
-    {
-      if (emit_csv)
-      {
-        fields.push_back(it->name);
-        fields.push_back(it->dsorg);
-        fields.push_back(it->volser);
-        std::cout << zut_format_as_csv(fields) << std::endl;
-        fields.clear();
-      }
-      else
-      {
-        std::cout << left << setw(44) << it->name << " " << it->volser << " " << it->dsorg << endl;
-      }
-    }
```

Comment:
quick Q: was this moved elsewhere? just double checking as we'll need the CSV logic for parsing output in the middleware

Reply:
lol, this is why we do code reviews... will add this back soon

Reply:
ok I'm crazy... it's actually still there if you view the entire file. I had some code duplication which is also fixed by this PR

Reply:
ahh I see - GitHub's diffs are misleading. sorry about that, thanks for clarifying